lib/server.js

```diff
@@ -110,6 +110,31 @@ function create(log, error, config, routes, db) {
 
   server.connection(connectionOptions)
 
+  if (config.hpkpConfig && config.hpkpConfig.enabled) {
+    var hpkpOptions = {
+      maxAge: config.hpkpConfig.maxAge,
+      sha256s: config.hpkpConfig.sha256s,
+      includeSubdomains: config.hpkpConfig.includeSubDomains
+    }
+
+    if(config.hpkpConfig.reportUri){
+      hpkpOptions.reportUri = config.hpkpConfig.reportUri
+    }
+
+    if(config.hpkpConfig.reportOnly){
+      hpkpOptions.reportOnly = config.hpkpConfig.reportOnly
+    }
+
+    server.register({
+      register: require('hapi-hpkp'),
+      options: hpkpOptions
+    }, function (err) {
+      if (err) {
+        log.error('Failed to load plugin:', err)
```

Comment:
Is this where the `sha256s` validation error ends up? Should we also `process.exit` in that case or is that overkill?